packages/common-functions/test/coerce-array.test.js

```diff
@@ -25,7 +25,7 @@ describe('coerce array', () => {
     expect(coerceArray(null)).toEqual(undefined);
   });
 
-  it('should return an empty array if input is null', () => {
+  it('should return undefined if input is empty array', () => {
```

Comment:
shouldn't a coerced empty array still result in an empty array?

Reply:
no, there are only two settings - undefined, or an array with elements